NEWS.rst

```diff
@@ -1,6 +1,11 @@
 History
 =======
 
+0.13 (2023-03-14)
+-----------------
+* Upgrade to redis-py>=4.2 and fakeredis>=2, fix test warnings (#132)
+* This now requires at least Python 3.6 and no more aioredis
```

Comment:
Since it's not possible to run the tests on Python 3.6, I wouldn't even try to claim support for it (given that it's EOL).

Reply:
I'd suggest `>= 3.7`. If it turns out that 0.13 doesn't work right on Python 3.6, there is no easy way to retroactively tell pip that it should prefer 0.12 for Python 3.6.

Reply:
We have quite a spread of minimum versions for telstate users:

- katsdpcal: >= 3.5
- katdal: >= 3.6
- katsdpimager: >= 3.6
- katsdpcontroller: >= 3.8
- katsdpingest: unknown

But I guess 3.7 is a good minimum nowadays.